project/src/components/main-screen/main-screen.jsx

```diff
@@ -1,32 +1,38 @@
 import React, {Fragment} from 'react';
 import PropTypes from 'prop-types';
 
+import Logo from '../logo/logo';
+import GenreList from '../genre-list/genre-list';
 import SmallFilmCard from '../small-film-card/small-film-card';
 
-function MainScreen({promoFilm, films}) {
-  const {title, genre, year, poster, fullPoster} = promoFilm;
-  const posterAlt = `${title} poster`;
+const setPosterAlt = (name) => `${name} poster`;
 
+const getSmallFilmCard = ({id, name, posterImage}) => (
+  <SmallFilmCard
+    key={id}
+    name={name}
+    posterImage={posterImage}
+  />
+);
+
+function MainScreen({
+  promoFilm: {name, posterImage, backgroundImage, genre, released},
+  films,
+}) {
   return (
     <Fragment>
```

Comment:
```suggestion
    <>
```

Fragment явно не принято объявлять, если только там не нужен key